src/main/java/commands/InsertMessageCommand.java

```diff
@@ -0,0 +1,70 @@
+package commands;
+
+import java.io.DataOutputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+
+import com.arangodb.ArangoCollection;
+import com.arangodb.ArangoDB;
+
+import database.ArangoChatHandler;
+import models.Command;
+import models.Message;
+import redis.clients.jedis.Jedis;
+
+public class InsertMessageCommand extends Command {
+	int numberOfMessages = 0;
+	ArrayList<Message> allMessages = new ArrayList<Message>();
```

Comment:
What does this command do? If doing batch inserts shouldn't be handled here, it should be in the chat server, and it calls a command with the complete batch of messages.